Automattic/wp-calypso
# Add Stats to cloud.jetpack.com/pricing

Context: p1HpG7-nWn-p2#comment-65153

## Proposed Changes

* Update the pricing page to include Stats (not only on the post-connection pricing page)
* Update the lightbox info for Stats:
  * Add benefits included
  * Add features included
  * Add "Great for" content
* Update description, short description, and display names
* Remove hardcoded pricing

<img width="1217" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/098ca28e-6378-4867-b208-6d340c0b40ce">


## Testing Instructions



* Open Jetpack Cloud live link
* Visit the pricing page `/pricing`
* Confirm that Stats is present and matches copy in the document linked above
* Test that pricing is correct and that you are able to complete the purchase of the product
* Extra: Open Calypso live link and `visit /stats/purchase`, select a site and play with going to the cart through the decision path for Stats. Make sure it redirects the same way it's currently doing on the production code

## Pre-merge Checklist
